### ISSUE TITLE
Read file from the end

### ISSUE THREAD

**Issue by Pt32** [greiman/SdFat#393]
Hi,

I want to read a file from the end with a buffer[24].
I use seek() to go back in the file. 

That's work perfectly. 

But it's aproximatly 10x longer than using read() to read the file from the top. 

Is there an another way to be more efficient ? And took the same time that read() function. 
Do I need to recode the read() function?

Thanks,

**Comment by greiman** [comment]
No, FAT file systems and SD cards are not designed to read from the end.  You could improve the speed by using a very large buffer.
